sphinxcontrib/openapi/renderers/_httpdomain.py

```diff
@@ -1,10 +1,9 @@
 """OpenAPI spec renderer."""
 
 import sys
-if sys.version_info >= (3, 3):
-    import collections.abc as collections
-else:
-    import collections
+
+import collections
+import collections.abc as collections
```

Comment:
So this doesn't work for me, I get the following error:

```
Exception occurred:
  File "/usr/local/lib/python3.10/site-packages/sphinxcontrib/openapi/renderers/_httpdomain.py", line 181, in <module>
    [(collections.abc.Mapping, deepmerge.strategy.dict.DictStrategies("merge"))],
AttributeError: module 'collections.abc' has no attribute 'abc'
```
Maybe using a distinct alias, something like this? (I haven't tested)

```suggestion

import collections
if sys.version_info >= (3, 3):
    import collections.abc as abc
else:
    import collections as abc
```

(On a quick local test, a double import of a package seems to work, although I don't know what the best practice would be).

Reply:
I tested the above [on that branch](https://github.com/qmonnet/openapi/commit/bebef13da8a1e3248ce36ad81e17d790095ee47c), and it works for me. Let me know how it goes in your case.

Reply:
Indeed, my code was wrong, I forgot to remove the `as collections` part behind the import. I pushed a new version, PTAL.